readthedocs/doc_builder/python_environments.py

```diff
@@ -87,6 +98,70 @@ def venv_bin(self, filename=None):
             parts.append(filename)
         return os.path.join(*parts)
 
+    def environment_json_path(self):
+        """Return the path to the ``environment.json`` file for this venv."""
+        return os.path.join(
+            self.venv_path(),
+            'environment.json',
+        )
+
+    @property
+    def is_obsolete(self):
+        """
+        Determine if the Python version of the venv obsolete.
+
+        It checks the the data stored at ``environment.json`` and compares it
+        against the Python version in the project version to be built and the
+        Docker image used to create the venv against the one in the project
+        version config.
+
+        :returns: ``True`` when it's obsolete and ``False`` otherwise
+
+        :rtype: bool
+        """
+        # Always return True if we don't have information about what Python
```

Comment:
This returns False.

Reply:
Ha! I did it wrong at first, then I fix the `return` statement and forgot the comment :)